libexec/std/internal/output.bash

```diff
@@ -0,0 +1,251 @@
+#
+# output.bash
+# -----------
+#
+# Private functions implementing output formatting. Used by public helper
+# functions.
+#
+
+
+# Print an error message to the standard error. The message is specified either
+# by positional parameters or on the standard input (by piping or redirection).
+#
+# Globals:
+#   none
+# Arguments:
+#   $@ - [opt = STDIN] message to print
+# Returns:
+#   none
+# Inputs:
+#   STDIN - [opt = $@] message to print
+# Outputs:
+#   STDERR - error message
+_err () {
```

Comment:
I added the `batslib_` prefix. I think that's more descriptive.

Fixed the formatting of function headers, and reformatted function comments so they are 72 character wide at most.